src/package.json

```diff
@@ -24,8 +24,7 @@
   "homepage": "https://github.com/single-spa/single-spa-angular#readme",
   "sideEffects": false,
   "peerDependencies": {
-    "@angular/core": ">=9",
-    "single-spa": ">=4.0.0"
```

Comment:
This needs to be moved into the `dependencies` field within this package.json. I only see it removed from the package.json and then added to the yarn.lock. The yarn.lock addition is not sufficient, since npm/yarn install based on package.json

Reply:
Alright, I moved it to the parent package.json: https://github.com/single-spa/single-spa-angular/pull/190/files

Will update the PR today and see how it goes.